server/cache_test.go

```diff
@@ -531,7 +533,8 @@ func (s *testClusterInfoSuite) testRegionSplitAndMerge(c *C, cache *clusterInfo)
 }
 
 func (s *testClusterInfoSuite) TestUpdateStorePendingPeerCount(c *C) {
-	cluster := newClusterInfo(core.NewMockIDAllocator())
+	_, opt := newTestScheduleConfig()
+	cluster := newClusterInfo(core.NewMockIDAllocator(), opt, core.NewKV(core.NewMemoryKV()))
```

Comment:
I have noticed this use pattern appeared too many times in tests, consider to extract a function like 'newMockClusterInfo'.